.changeset/smart-mice-crash.md

```diff
@@ -0,0 +1,6 @@
+---
+'@crowdstrike/ember-toucan-core': minor
+'@crowdstrike/ember-toucan-form': minor
+---
+
+Removed from Autocomplete support for `@options` as an array of objects.
```

Comment:
I'd expand on this and add examples, but Autocomplete hasn't been released yet. So no need, as far as I can tell, to give consumers backstory, reasoning, or a migration path.